wolfcrypt/src/integer.c

```diff
@@ -3177,7 +3177,7 @@ int fast_s_mp_mul_digs (mp_int * a, mp_int * b, mp_int * c, int digs)
   {
     mp_digit *tmpc;
     tmpc = c->dp;
-    for (ix = 0; ix < pa+1; ix++) {
+    for (ix = 0; ix < pa; ix++) { /* JRB, +1 could read uninitialized data */
```

Comment:
I'm not positive what the original purpose of the +1, but perhaps it would make better sense to adjust the `if (pa > MP_WARRAY)` check above at line 3131 to `if (pa+1 > MP_WARRAY)`?